habitat_baselines/common/trainer_registry.py

```diff
@@ -0,0 +1,27 @@
+from typing import Optional
+
+from habitat.core.registry import Registry
+
+
+class TrainRegistry(Registry):
```

Comment:
Why create a new registry class? Whole point of Registry pattern is to create a singleton which is easy to maintain instead of multiple sources of global truth. I would suggest using better functionalities in Registry itself to encapsulate everything there. For reference, check Pythia's registry again.